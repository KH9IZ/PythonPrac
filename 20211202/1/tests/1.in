class My(metaclass=dump):
    def __call__(self, qq):
        print(qq.__repr__())
    tmp = lambda self: "smthg"
    val = 0

my = My()
my('123')
my.tmp()
print(my.val)
