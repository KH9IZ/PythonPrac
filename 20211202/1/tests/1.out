__call__: ('123',), {}
'123'
tmp: (), {}
0
